### ISSUE TITLE
EXPLOIT - Being Small (Dwarf) Makes the Psychic Glow Nearly Invisible

### ISSUE THREAD

**Issue by KasparoVy** [ParadiseSS13/Paradise#2397]
Having the dwarf mutation while having the psychic mutation eliminated the blue head-glow in an un-replicable circumstance, making it impossible to identify people using this power.
Otherwise, being a dwarf and having psychic reduces the visibility of the glow to the point where it is nearly absent and easy to obscure
Happened after #2377 was merged

It's already been used to grief a couple times and avoid detection (only by other players, thankfully)

**Comment by tigercat2000** [comment]
@Fox-McCloud :eyeglasses:

**Comment by Fox-McCloud** [comment]
No idea how to fix this---the TK stuff is an overlay, and I've largely not played with overlays or how they interact with mob sprite resizing.

**Comment by KasparoVy** [comment]
I find it weird that 'shrinking' does this-- does it put the shrunk sprite on a layer higher than that of the one TK uses maybe?

**Comment by veganzombeh** [comment]
That's weird. Resize is just a transform, pretty much the same as lying. Do you happen to know if this is an issue for lying too?

**Comment by KasparoVy** [comment]
Nope, but I'll test it right now and get back to you.

**Comment by KasparoVy** [comment]
![psychic](https://cloud.githubusercontent.com/assets/12377767/10624469/8f9a1d3a-7766-11e5-8cd8-291654effb34.png)

Testing on local yielded this result.
As seen on the teeny tiny captain, the glowing halo is barely visible and extremely easy to obscure.

However, I am absolutely certain that in the circumstances that provoked me to write this bug report, there was no such halo visible-- and the person who was doing it was, indeed, tiny

Guess all it warrants is an increase in visibility if anything, I can't reproduce the halo being straight-up invisible on local

I'll rewrite the main post now

**Comment by KasparoVy** [comment]
The issue is still very real, and the same as I first thought. On live, the halo doesn't even pop up. Here's a screenie: 
![bugerino](https://cloud.githubusercontent.com/assets/12377767/10708584/c1e64d0a-79db-11e5-958b-22de77a6db3a.PNG)


This also happened before I got shrunk. When I gave the RD an injector of TK, they didn't get the glow neither. Same with a Vox I gave it to.
BIG BAD BUG BLOW MA HOUSE DOWN
